src/components/common/PageSearch/PageSearch.tsx

```diff
@@ -0,0 +1,79 @@
+import ArrowDownwardIcon from '@mui/icons-material/ArrowDownward'
+import ArrowUpwardIcon from '@mui/icons-material/ArrowUpward'
+import SearchIcon from '@mui/icons-material/Search'
+import { Box, IconButton, InputAdornment, TextField } from '@mui/material'
+import { observer } from 'mobx-react-lite'
+import React, { ChangeEvent, useEffect, useState } from 'react'
+
+import { useSharedStore } from '../../SharedStoreProvider'
+
+import { useStyles } from './PageSearch.styles'
+
+const PageSearchInput = () => {
```

Comment:
why not `export const PageSearch = observer(() => {`

Reply:
👍